app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragmentPresenter.kt

```diff
@@ -186,13 +185,4 @@ class TopicLessonsFragmentPresenter @Inject constructor(
   fun storySummaryClicked(storySummary: StorySummary) {
```

Comment:
Not able to comment on that particular line but writing here. 
There is an unused import at line number 15 `import org.oppia.app.model.EventLog`

Reply:
Done.